modules/eco-node-health-check-operator-about.adoc

```diff
@@ -61,4 +61,33 @@ During the upgrade process, nodes in the cluster might become temporarily unavai
 When a node is identified as unhealthy, the Operator checks how many other nodes are unhealthy. If the number of healthy nodes exceeds the amount that is specified in the `minHealthy` field of the `NodeHealthCheck` CR, the controller creates a remediation CR from the details that are provided in the external remediation template by the remediation provider. After remediation, the node's health status is updated accordingly.
 
 When the node turns healthy, the controller deletes the external remediation template
-and updates the node's health status.
\ No newline at end of file
+and updates the node's health status.
+
+[id="understanding-nhc-behavior-with-mhc_{context}"]
+== Understanding node health checks behavior with machine health checks
+
+When both, node health checks and machine health checks are deployed, the node health checks avoid conflicts with the machine health checks.
+
+The following list summarises the system behaviour when node health checks and machine health checks are deployed:
+
+* If only the default machine health check exists, the node health check continues to identify unhealthy nodes. Unhealthy nodes in a `Terminating` condition are ignored.
```

Comment:
Unhealthy nodes in a `Terminating` condition are ignored and will be handled by machine health check.

Reply:
(just a suggestion)